src/format_data.py

```diff
@@ -0,0 +1,37 @@
+from keras.utils import np_utils 
+import numpy as np
+import pandas as pd
+
+raw_data_csv_file_name = '../../data/fer2013.csv'
```

Comment:
This should probably be `'../data/fer2013.csv'`, given the latest file structure.